frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/network-attachment-definition.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { sortable } from '@patternfly/react-table';
+import { ListPage, Table, TableData, TableRow } from '@console/internal/components/factory';
+import { Kebab, ResourceKebab, ResourceLink } from '@console/internal/components/utils';
+import { NamespaceModel } from '@console/internal/models';
+import { K8sResourceKindReference } from '@console/internal/module/k8s';
+import { getName, getNamespace, getUID } from '@console/shared/src';
+import { getConfig } from '../../utils';
+import { NetworkAttachmentDefinitionModel } from '../../models';
+import { NetworkAttachmentDefinitionKind } from '../../types';
+
+const { common } = Kebab.factory;
+const menuActions = [...common];
+
+export const NetworkAttachmentDefinitionReference: K8sResourceKindReference =
+  'NetworkAttachmentDefinition';
+
+const tableColumnClasses = [
+  classNames('col-lg-4', 'col-md-4', 'col-sm-6', 'col-xs-6'),
+  classNames('col-lg-4', 'col-md-4', 'hidden-sm', 'hidden-xs'),
+  classNames('col-lg-4', 'col-md-4', 'col-sm-6', 'col-xs-6'),
+  Kebab.columnClass,
+];
+
+const NetworkAttachmentDefinitionsHeader = () => {
+  return [
```

Comment:
you can extract `dimensifyHeader` and `dimensifyRow` ([usage](https://github.com/openshift/console/blob/master/frontend/packages/kubevirt-plugin/src/components/vms/vm.tsx)) from `kubevirt-plugin` to `console/shared` if you want; for column classes easier resolution and column indexes runtime control.

Reply:
Done!